src/Instrumentation/Symfony/src/MessengerInstrumentation.php

```diff
@@ -125,7 +125,7 @@ public static function register(): void
                 $builder = $instrumentation
                     ->tracer()
                     ->spanBuilder(\sprintf('SEND %s', $messageClass))
-                    ->setSpanKind(SpanKind::KIND_INTERNAL)
```

Comment:
I'm no expert here, but aren't dispatch and send roughly the same thing (ie, outbound messages)? I would have thought that KIND_CONSUMER would be for receiving messages?

Reply:
Thank you for the feedback, @brettmc ! You’re right that both dispatch and send in Symfony Messenger handle outbound messages.

Regarding the use of KIND_CONSUMER, you bring up a good point. In OpenTelemetry's messaging semantic conventions, KIND_PRODUCER is typically used for the sender (dispatch or send), and KIND_CONSUMER is used for receiving messages.

Given that this instrumentation focuses on tracing the message dispatch and sending processes (outbound), I guess the correct span kind would be KIND_PRODUCER for both events right?.

Reply:
Is this resolved? It's still KIND_CONSUMER.

Reply:
Oh, surprised that didn't update, thought I pushed already. Updated now

Reply:
I think you need to update a test as well...

Reply:
Test updated